scripts/dockers/cb_master/configure-portal.py

```diff
@@ -254,10 +255,23 @@ def patch_mail_client(self):
             mailclienthrd.set('instance.smtp.%s' % key, value)
         mailclienthrd.save()
 
+
+    def configure_manifest(self):
+        with open('/opt/cfg/system/versions-manifest.yaml') as file_discriptor:
+            data_str = file_discriptor.read()
+            data_obj = yaml.load(data_str)
+        version = self.scl.version.new()
```

Comment:
this is not omnipotent...
check if current version already exists in db first...
Also who is going to change the status from isntalling to CURRENT?